src/tasks/tasks.controller.spec.ts

```diff
@@ -1,18 +1,100 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { TasksController } from './tasks.controller';
+import { TasksService } from './tasks.service';
+import { TasksRepository } from './tasks.repository';
+import { v4 as uuidv4 } from 'uuid';
+
+// jest.mock('./tasks.service');
 
 describe('TasksController', () => {
-  let controller: TasksController;
+  let tasksController: TasksController;
+  let tasksService: TasksService;
+
+  const mockUserService = {
+    create: jest.fn((dto) => {
+      return {
+        taskId: uuidv4(),
+        ...dto,
+      };
+    }),
+    find: jest.fn(() => {
+      return [];
+    }),
+    findOneAndUpdate: jest.fn((taskId, dto) => {
+      return {
+        taskId: uuidv4(),
+        ...dto,
+      };
+    }),
+  };
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
       controllers: [TasksController],
+      providers: [
+        TasksService,
+        {
+          provide: TasksRepository,
+          useValue: mockUserService,
+        },
+      ],
     }).compile();
 
-    controller = module.get<TasksController>(TasksController);
+    tasksController = module.get<TasksController>(TasksController);
+    tasksService = module.get<TasksService>(TasksService);
   });
 
   it('should be defined', () => {
-    expect(controller).toBeDefined();
+    expect(tasksController).toBeDefined();
+  });
+
+  it('should create a user!', async () => {
+    try {
+      var actual = await tasksController.createTask({
+        email: 'testing123344@gmail.com',
+        age: 22,
+      });
+    } catch (err) {
+      console.log(err);
```

Comment:
I guess by default jest suppresses the console logs can you please put a screenshot of this over here?

Reply:
![image](https://user-images.githubusercontent.com/31640506/147550810-4b4fec6b-d91a-4341-a2ed-1ad43089c3ad.png)

Reply:
This is not what I asked for run the code and make sure it goes to the catch block I want to see the error up on console.